src/jvmTest/kotlin/test.kt

```diff
@@ -1,8 +1,11 @@
 import org.junit.jupiter.api.Test
```

Comment:
(this comment is for tests in general, not for this specific line)
Add description for new tests (cnf files) in you md file: specify the source of these CNFs, and estimate complexity (time/conflicts as solved by MiniSat, for example).